src/components/shared/DataTable.js

```diff
@@ -110,8 +111,8 @@ function DataTable({
 
   return (
     <ErrorBoundary onError={(err, componentStack) => addError({ error: { message: err.message, componentStack } })} FallbackComponent={ErrorFallback}>
-      <div className="react-table-scroller">
-        <DataTableHeader headerGroups={headerGroups} onSortedChange={onSortedChange} sorted={sorted} showFilter={showFilter} />
+      <div role="table" aria-label="Table Data" className="react-table-scroller">
+        <DataTableHeader headerGroups={headerGroups} onSortedChange={onSortedChange} sorted={sorted} showFilter={showFilter} dynamicAttributesFromDataTable={dynamicAttributesFromDataTable} />
```

Comment:
@BboyAkers a little off topic but i think you would benefit from adding a linter and a git hook.
github is wrapping these commits in a weird way and would be easier to read your changes if it was all formatted the same

* https://github.com/prettier/eslint-config-prettier#installation
* https://prettier.io/docs/en/install.html#git-hooks

Reply:
Thanks for the feedback!! I'll add one this week!! @cjoshmartin  😃